dev-tools/validation/check-authz-route-matrix.py

```
@@ -134,42 +2543,237 @@ def validate_known_drift(value: Any, field: str, errors: list[str]) -> None:
+        return None
+    service, separator, route = value.partition("/")
+    components = (
+        canonical_route_components(service, route) if separator else None
+    )
+    if components is None:
+        errors.append(f"{field} must be a non-empty service/route identity")
+        return None
+    return "/".join(components)
+
+
+def validate_operator_mutation_support_gate(
```

Ruff (0.16.0)

[warning] Too many branches (29 > 12)

(PLR0912)